src/components/layout/NavBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { Link } from "react-router-dom";
+
+const NavBar = () => {
+  return (
+    <nav className="bg-background-default py-2 border-t border-background-light">
+      <ul className="flex justify-center space-x-8">
+        <li>
+          <Link to="/vinyls" className="text-text-primary hover:text-primary">
+            VINYLS
+          </Link>
+        </li>
+        <li>
+          <Link to="/genres" className="text-text-primary hover:text-primary">
+            GENRES
+          </Link>
+        </li>
+        <li>
+          <Link to="/artists" className="text-text-primary hover:text-primary">
+            ARTISTS
+          </Link>
+        </li>
+        <li>
+          <Link to="/labels" className="text-text-primary hover:text-primary">
+            LABELS
+          </Link>
+        </li>
+        <li>
+          <Link to="/new-releases" className="text-text-primary hover:text-primary">
+            NEW RELEASES
+          </Link>
+        </li>
+        <li>
+          <Link to="/on-sale"className="text-text-primary hover:text-primary">
+            ON SALE
+          </Link>
+        </li>
+        <li>
+          <Link to="/shipping" className="text-text-primary hover:text-primary">
```

Comment:
considering that the exact same styling is being applied to each Link component here perhaps you can use the same method that you used in Button.tsx and save the styling string inside a variable at the top and call it in each instance here. That way if we wanted to change it we only need to do so in the variable at the top to apply to all instances.

Reply:
Hey Max, thanks for pointing that out! You're right, I hadn't thought about using a variable to store the styles. I’ve now refactored the NavBar component to use the linkStyles variable. This makes it much easier to manage and update in the future.